backend/data/src/main/kotlin/io/tolgee/service/organization/OrganizationService.kt

```diff
@@ -80,7 +80,7 @@ class OrganizationService(
     userAccount: UserAccount,
   ): Organization {
     if (createDto.slug != null && !validateSlugUniqueness(createDto.slug!!)) {
-      throw ValidationException(Message.ADDRESS_PART_NOT_UNIQUE)
+      throw ValidationException(Message.SLUG_NOT_UNIQUE)
```

Comment:
Is the `ADDRESS_PART_NOT_UNIQUE` used anywhere now? I may be deleted also.

Reply:
The constant was actually renamed.